### ISSUE TITLE
Improvement for k8s.io/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/

### ISSUE THREAD

**Issue by fabpico** [kubernetes/website#43258]
Avoid abbrevations

> A **PersistentVolume (PV)** is a piece of storage in the cluster that has been manually provisioned by an administrator, or dynamically provisioned by Kubernetes using a StorageClass. A **PersistentVolumeClaim (PVC)** is a request for storage by a user that can be fulfilled by a **PV. PersistentVolumes** and PersistentVolumeClaims are independent from Pod lifecycles and preserve data through restarting, rescheduling, and even deleting Pods.

When a sentence ends with an abbrevation, and the next sentence starts with the expanded word of the same abbrevation  
`.. PV. PersistentVolumes ..`, that sounds a bit weird.

> Note: It can take up to a few minutes for the **PV**s to be provisioned and bound.

I would find it more clear just to write PersistentVolumes.

While we're at it, 'Polyvinyl Chloride' (PVC) can also be left out.

**Comment by dipesh-rawat** [comment]
_Page related to issue:_ https://kubernetes.io/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/
/language en

**Comment by dipesh-rawat** [comment]
/retitle Improve clarity by avoiding abbreviations in descriptions of PersistentVolumes and PersistentVolumeClaims
/sig docs

**Comment by Affan-7** [comment]
/assign

I would like to work on this.
Waiting for triage.

**Comment by tengqm** [comment]
I don't think this worth a fix. When the acronyms (PV, PVC) first appeared on this page, they were explained. Later on this page (and many other pages), we use "PV" and "PersistentVolume" interchangeably. It is like calling "US" and "United States".
Feel free to reopen this if people really feel that this hurts readability.
/close

**Comment by fabpico** [comment]
> It is like calling "US" and "United States".

Still an abbrevation, but I find Kubernetes abbreviations should not be equated with globally established abbreviations like 'US'. There are already more than enough abbreviations that we need to know. And I find this should slowly stop. I find more mental mapping should be avoided.
